bot/cogs/jams.py

```diff
@@ -78,6 +96,14 @@ async def createteam(self, ctx: commands.Context, team_name: str, members: comma
                 connect=True
             )
 
+        return team_channel_overwrites
+
+    async def create_channels(self, ctx: commands.Context, team_name: str, members: t.List[Member]) -> str:
+        """Create team text and voice channel. Return name of text channel."""
```

Comment:
This returns the mention, not the name.
```suggestion
        """Create team text and voice channels. Return the mention for the text channel."""
```